hi-ml-cpath/src/health_cpath/preprocessing/create_tiles_dataset.py

```diff
@@ -292,11 +295,11 @@ def main(slides_dataset: SlidesDataset, root_output_dir: Union[str, Path],
 
 
 if __name__ == '__main__':
-    from health_cpath.datasets.tcga_prad_dataset import TcgaPradDataset
+    from cpath.datasets.tcga_prad_private_dataset import TcgaPradPrivateDataset
 
     # Example set up for an existing slides dataset:
-    main(slides_dataset=TcgaPradDataset("/tmp/datasets/TCGA-PRAD"),
-         root_output_dir="/datasetdrive/TCGA-PRAD_10X_tiles_level1_224",
+    main(slides_dataset=TcgaPradPrivateDataset("/panda_dataset/TCGA-PRAD_20220712"),
```

Comment:
If I'm not mistaken, there is no class `TcgaPradPrivateDataset` here?

Reply:
True, we need to use `TcgaPradDataset` here